src/main/groovy/JsJobBuilder.groovy

```diff
@@ -37,12 +49,14 @@ class JsJobBuilder {
                         '''
                 )
             }
+
             publishers {
                 archiveArtifacts artifacts
-                if (emails) {
-                    mailer emails.join(' ')
-                }
             }
+
         }
+
+        baseJob
```

Comment:
is this implicit return actually needed?  I had thought calling `baseJob.with` would be sufficient for what to return, but correct me if I'm wrong.